src/styles/index.ts

```diff
@@ -3674,6 +3674,22 @@ const styles = (theme: ThemeColors) =>
             ...wordBreak.breakWord,
         },
 
+        quickActionTooltipWrapper: {
+            backgroundColor: colors.green100,
+        },
+
+        quickActionTooltipTitle: {
+            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE_BOLD,
+            fontWeight: FontUtils.fontWeight.bold,
+            fontSize: variables.fontSizeNormal,
```

Comment:
cc @dubielzyk-expensify in case this should be `label` and not `normal`.

Reply:
Yep should be `label`